restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/mybatis/MybatisApplicationsDao.java

```diff
@@ -94,6 +95,39 @@ private Application getApplication(final String selector, final String parameter
     }
 
     @Override
+    public List<Application> getApplications(Sid accountSid, boolean includeNumbers) {
+        if (!includeNumbers)
+            return getApplications(accountSid);
+
+        final SqlSession session = sessions.openSession();
+        try {
+            final List<Map<String, Object>> results = session.selectList(namespace + "getApplicationsAndNumbers", accountSid.toString());
```

Comment:
again, the method with boolean flag, and the sql query involved doesnt make much sense. "getApplicationsAndNumbers" sql query is always invoked regardless the boolean flag

Reply:
ok, agreed.